extension/serializer/src/main/resources/avro/MetaData-v1.avsc

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "record",
+  "namespace": "io.holixon.axon.avro.serializer.metadata",
+  "name": "AvroMetaData",
+  "doc": "axon message MetaData, see org.axonframework.messaging.MetaData",
+  "revision": "1",
+  "fields": [
+    {
+      "name": "values",
```

Comment:
missing doc

Reply:
you mean the "doc" field?

Reply:
It is not only doc field - but this gets generated in the event class too. Let's make this a tradition to EVER create full documented APIs / Schemas :-)

Reply:
json does not support docs/comments and the class is generated ... open for suggestions how this should be documented

Reply:
I mean the "doc" field on every field... Look here: https://github.com/holixon/avro-registry-adapter/blob/4d5c111781854385b04976da968b9ce945cb84e8/extension/registry/holixon/holixon-axon/src/main/avro/holixon.registry.AvroSchemaRegistered-v1.avsc#L26